forte/__init__.py

```diff
@@ -59,3 +59,14 @@
     psi_options = psi4.core.get_options()
     psi_options.set_current_module('FORTE')
     forte_options.push_options_to_psi4(psi_options)
+
+    # Set options to psi4 values
+    forte_options.add_double("INTS_TOLERANCE",
+                             psi_options.get_double("INTS_TOLERANCE"),
+                             "Schwarz screening threshold")
+    forte_options.add_double("DF_FITTING_CONDITION",
+                             psi_options.get_double("DF_FITTING_CONDITION"),
+                             "Threshold for DF fitting condition")
+    forte_options.add_double("CHOLESKY_TOLERANCE",
+                             psi_options.get_double("CHOLESKY_TOLERANCE"),
+                             "The tolerance for Cholesky integrals")
```

Comment:
@fevangelista Sorry for the repeated messages. I finally moved `INTS_TOLERANCE` and `CHOLESKY_TOLERANCE` from `register_forte_options.py` to here due to an issue in future python model. I think they'd better be consistent with Psi4 values by default. I can move all of them back to where they were if you find inconvenient or if they break the coding style.

Reply:
I ended up follow the convention. However, there is a more organized verison in commit 51a97d5.